src/modules/job-manager/job-manager.c

```diff
@@ -61,6 +61,24 @@ void disconnect_rpc (flux_t *h,
     event_listeners_disconnect_rpc (h, mh, msg, arg);
 }
 
+static void stats_cb (flux_t *h, flux_msg_handler_t *mh,
+                      const flux_msg_t *msg, void *arg)
+{
+    struct job_manager *ctx = arg;
+    int listeners = event_listeners_count (ctx->event);
+    if (flux_respond_pack (h, msg, "{s:i}",
+                           "listeners", listeners) < 0) {
+        flux_log_error (h, "%s: flux_respond_pack", __FUNCTION__);
```

Comment:
Maybe change the key to "event-listeners" in case we add another streaming RPC to job manager that we want to count here.